roles/mysql_hardening/tasks/mysql_secure_installation.yml

```diff
@@ -37,58 +37,51 @@
 - name: Ensure that root can only login from localhost
   community.mysql.mysql_query:
     query:
-      - DELETE
-        FROM mysql.user
-        WHERE USER='root'
-          AND HOST NOT IN ('localhost',
-                          '127.0.0.1',
-                          '::1')
+      # yamllint disable-line rule:line-length
+      - DELETE FROM mysql.user WHERE USER='root' AND HOST NOT IN ('localhost', '127.0.0.1', '::1')
     login_unix_socket: "{{ login_unix_socket | default(omit) }}"
   when: mysql_remove_remote_root
 
+# yamllint disable-line rule:line-length
 - name: Get all users that have no authentication_string on MySQL version >= 5.7.6 or Mariadb version >= 10.4.0
   community.mysql.mysql_query:
     query:
-      - SELECT GROUP_CONCAT(QUOTE(USER), '@', QUOTE(HOST) SEPARATOR ', ') AS users
```

Comment:
This might be debatable, but I'm in the camp "if my linter does is, I accept it."

Reply:
I'm ok with that